frontend/src/store/GrantAccessDataState.ts

```diff
@@ -1,35 +1,30 @@
 import { ref } from 'vue';
-import type { FamUserRoleAssignmentCreate } from 'fam-api';
+import type { FamForestClient, FamUserRoleAssignmentCreate } from 'fam-api';
 
 export const domainOptions = { IDIR: 'I', BCEID: 'B' }; // TODO, load it from backend when backend has the endpoint.
 
-export const grantAccessFormData = ref<FamUserRoleAssignmentCreate | null>();
+export const grantAccessFormData = ref<FamUserRoleAssignment | null>();
 export const grantAccessFormRoleName = ref<string | null>();
 
 export const FOREST_CLIENT_INPUT_MAX_LENGTH = 8;
 
-export const setGrantAccessFormData = (formData: any) => {
+export const setGrantAccessFormData = (formData: any, forestClientList: any) => {
```

Comment:
forestClientList should have a type

Reply:
Right, it should be an array of FamForestClient. I'll correct it. Thanks.